gulpfile.js

```diff
@@ -21,7 +21,7 @@ gulp.task('nsp', function (cb) {
 });
 
 gulp.task('pre-test', function () {
-  return gulp.src(['**/*.js', '!**/templates/**'])
+  return gulp.src(['generators/**/*.js', '!**/templates/**'])
```

Comment:
We actually want to cover every js files in here. It's useful to know when there's dead code in test code, etc.